invenio_theme/assets/semantic-ui/js/invenio_theme/theme.js

```diff
@@ -12,12 +12,13 @@ import "./truncate.js";
 import "semantic-ui-css/semantic.js";
 import "semantic-ui-less/semantic.less";
 
-function imagePlaceholder() {
-  this.onerror=null;
-  this.src='/static/images/square-placeholder.png';
-}
-
-jquery(".has-placeholder").attr('onerror', imagePlaceholder);
+// Add placeholder image if image not found
+jquery(".has-placeholder").on('error', function() {
```

Comment:
agreed on one of the stand-ups to replace with the opposite workflow:
1) load and display placeholder (the loading placeholder from semantic, not the static one)
2) check if the image exists
3) if yes replace the placeholder with the image
4) if no, display static empty placeholder image (/static/images/square-placeholder.png) not the same as the loading one